emperor/flat_plots.py

```diff
@@ -0,0 +1,346 @@
+from matplotlib import use
+use('Agg')  # noqa
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+
+def plot_3x3(ordination, offset, mapping, field, colors, null_kw=None,
```

Comment:
@ElDeveloper, `plot_3x3` is just one possible layout, and is one which I am currently using and which fits my usecase well. The underlying methods to produce individual plots are exposed; other layout support would be great but in my opinion, it is out of scope for this PR.